docs/products/postgresql/howto/monitor-pgbouncer-with-datadog.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Monitor PgBouncer with Datadog
+sidebar_label: Monitor PgBouncer in Datadog
+---
+
+Integrate PgBouncer with Datadog to track connection pool metrics and monitor application traffic on the Datadog platform.
+
+## Prerequisites
+
+- Your service plan is Startup or higher.
+- You applied outstanding service maintenance updates affecting the Datadog Metrics
+  integration.
+
+## Enable monitoring
+
+You can enable monitoring PgBouncer metrics for your Aiven for PostgreSQL service both
+[if the service already has a Datadog Metrics integration](#enable-monitoring-if-the-integration-exists)
+and [if it doesn't](#create-the-integration-and-enable-monitoring).
+
+### Enable monitoring for an integrated service{#enable-monitoring-if-the-integration-exists}
+
+Enable monitoring PgBouncer metrics for an Aiven for PostgreSQL service that already
+has a Datadog Metrics integration:
+
+1. Obtain the SERVICE_INTEGRATION_ID of the Datadog Metrics integration for your Aiven for
```

Comment:
Can we put variable names as inline code to help with scanability (applies to the whole file)?

```suggestion
1. Obtain the `SERVICE_INTEGRATION_ID` of the Datadog Metrics integration for your Aiven for
```